pulp_rpm/app/constants.py

```diff
@@ -13,11 +13,6 @@
 
 # The same as above, but in a format that choice fields can use
 CHECKSUM_CHOICES = (
```

Comment:
this is also used on the Package model and CustomRepodata model, we probably would want to save packages with e.g sha224

Reply:
If any exist :)  But you're right...